.github/workflows/linux.yml

```
@@ -253,6 +253,22 @@ jobs:
         cmake --build build --parallel 2
         ctest --test-dir build --output-on-failure
 
+        python3 -m pip install jsonschema
```

Extra section might be cleaner:
```suggestion
    - name: Schema Check
      run: |
        python3 -m pip install jsonschema
```